modules/swagger-codegen/src/main/resources/TypeScript-Fetch/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Don't we need to have an install script to do tsc and install typings? see https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/typescript-angular2/package.mustache for an example.